packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -5,7 +5,7 @@ excerpt: Customize tokens returned by the Okta API Access Management process flo
 
 # Token Inline Hook Reference
 
-This page provides reference documentation for:
+This page provides reference documentation for Token Inline Hooks.
```

Comment:
The original stem introduced the bulleted items right below. There is no connection now to what those bulleted items are telling you. The title is just above that stem sentence, was repeating the title intentional? I suggest we leave it as it was or try one of these two suggestions:

This page provides reference documentation for Token Inline Hooks. It provides:

OR

This page provides reference documentation for Token Inline Hooks. It provides sample JSON objects that are contained in the outbound request from Okta to your external service, and sample JSON objects that you can include in your response.

```suggestion
This page provides reference documentation for:
```